crates/bindings-csharp/Runtime/RawBindings.cs

```diff
@@ -63,28 +62,36 @@ public class StdbException : Exception
 
     public class NoSuchTableException : StdbException
     {
-        internal NoSuchTableException() : base(ERRNO_NO_SUCH_TABLE) { }
+        internal NoSuchTableException()
```

Comment:
These formatting changes are also in https://github.com/clockworklabs/SpacetimeDB/pull/1309